test/publicProfile.matchers.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const { _, definePattern } = require('./matchers');
+
+definePattern(
+  'aPublicProfile',
+  {
+    avatar: _.isAssetUrl,
+    banner: _.isUndefinedOr(_.isAssetUrl),
+    bio: _.isUndefinedOr(_.isString),
+    city: _.isUndefinedOr(_.isString),
+    country: _.isUndefinedOr(_.isString),
+    did: _.isDID,
+    displayName: _.isString,
+    postalCode: _.isUndefinedOr(_.isString),
+    province: _.isUndefinedOr(_.isString),
+    username: _.isString,
```

Comment:
there this could be a more specific and its own matcher

Reply:
see above